packages/flame/test/events/component_mixins/drag_callbacks_test.dart

```diff
@@ -131,10 +132,38 @@ void main() {
         expect(component.dragCancelEvent, 0);
       },
     );
+
+    testWithGame(
+      'make sure the FlameGame can registers DragCallback on itself',
+      _DragCallbacksGame.new,
+      (game) async {
+        await game.ready();
+        expect(game.children.length, 1);
+        expect(game.children.first, isA<MultiDragDispatcher>());
+      },
+    );
+
+    testWidgets(
+      'drag correctly registered handled event directly on FlameGame',
+      (tester) async {
+        final game = _DragCallbacksGame()..onGameResize(Vector2.all(300));
+        await tester.pumpWidget(GameWidget(game: game));
+        await tester.pump();
+        await tester.pump();
+        expect(game.children.length, 1);
+        expect(game.isMounted, true);
```

Comment:
```suggestion
        expect(game.children.length, equals(1));
        expect(game.isMounted, isTrue);
```

Reply:
The same suggestion to anywhere where a similar change would apply :)

Reply:
Fixed! We have a loooot of other tests to fix this in too.